README.md

```diff
@@ -54,6 +54,7 @@ Documentation of using RapidAPI can be found [here](https://docs.rapidapi.com/do
 
 ## Disclaimer
 
-Un-official chrome extension for Bionic Reading (BR®). All trademarks are the property of their respective owners.
+This project is not afflicted with Bionic Reading (BR®). All trademarks are the property of their respective owners.
```

Comment:
I think you might mean `this project is not affiliated with...` instead of afflicted

Reply:
I'v pushed a new fix resolving the typo.

Reply:
@fwzmhmd - I'v pushed a new fix resolving the typo.